src/components/phyx/PhyxView.vue

```diff
@@ -441,27 +440,26 @@ export default {
         ];
       });
 
-      // Convert to CSV.
+      // Convert to TSV.
       // console.log('Output:', [header, ...rows]);
-      stringify([
-        header,
-        ...rows,
-      ], (err, csv) => {
-        if (err) {
-          console.log('Error occurred while producing CSV:', err);
-          return;
-        }
+      const tsv = [];
 
-        const content = [csv];
-        // console.log('Content:', content);
+      function removeWhitespace(str) {
+        return str.replaceAll(/(?:\s|\n|\r\n)+/g, " ");
```

Comment:
I thought `\s` includes newlines. Have you found it doesn't in some circumstances?

Reply:
You are correct! I've replaced this with `str.replaceAll(/\s+/g, " ")` in 2e28eec.